### ISSUE TITLE
Problem with color settings on Rival 500

### ISSUE THREAD

**Issue by flozz** [flozz/rivalcfg#230]
Hi,
Just trying the software and found that -C indeed does not work on Rival 500.  -c or --logo-color changes both the logo and the wheel at the same time. 
Perhaps the color registers are packed or shifted and they must be both written at the same time?

_Originally posted by @ChrisDWhitehead in https://github.com/flozz/rivalcfg/issues/189#issuecomment-2071324237_

**Comment by flozz** [comment]
Hello,

Thank you for your repport, can you provide the output of the `rivalcfg --print-debug` command so we have the basic information about Rivalcfg, you system and the device?

I have a Rival 500 at home so I may be able to test/reproduce the issue as soon I find some time to work on this :)

**Comment by ChrisDWhitehead** [comment]
RIVALCFG
========
Version: 4.13.0
udev rules installed: True
udev rules up to date: True
Installation path: /usr/local/lib/python3.10/dist-packages/rivalcfg

OPERATING SYSTEM
================
OS: Linux
Platform: Linux-5.15.0-105-generic-x86_64-with-glibc2.35
Version: #115-Ubuntu SMP Mon Apr 15 09:52:04 UTC 2024
Distribution issue: Linux Mint 21.3 Virginia \n \l

PYTHON
======
Python version: 3.10.12
HIDAPI version: 0.14.0

PLUGGED STEELSERIES DEVICES ENDPOINTS
=====================================
1038:170e | 00 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
1038:170e | 01 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
1038:170e | 02 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)

**Comment by flozz** [comment]
> I apologize for the strange formatting, but apparently some of the formating of the output triggers some HTML formatting. The underlines are actually a string of '=' but it gets re-interpreted. I tried using code mode but it didn't do anything.

Yeah a part of the text was interpreted as Markdown syntax. You can put RAW content using 3 backtick:

<pre>
```
RAW CONTENT HERE
```
</pre>

**Comment by shawarden** [comment]
Has there been any progress on this? It is still happening.


```
rivalcfg --print-debug

RIVALCFG
========
Version: 4.17.0
udev rules installed: True
udev rules up to date: True
Installation path: /usr/lib/python3.14/site-packages/rivalcfg

OPERATING SYSTEM
================
OS: Linux
Platform: Linux-7.1.3-2-cachyos-x86_64-with-glibc2.43
Version: #1 SMP PREEMPT_DYNAMIC Wed, 08 Jul 2026 18:34:01 +0000
Distribution issue: \S{PRETTY_NAME} \r (\l)

PYTHON
======
Python version: 3.14.6
HIDAPI version: 0.14.0

PLUGGED STEELSERIES DEVICES ENDPOINTS
=====================================
1038:170e | 00 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
1038:170e | 01 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
1038:170e | 02 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
1038:170e | 02 | SteelSeries Rival 500 Gaming Mouse (firmware v101.0)
```
